LICENSE.txt

```diff
@@ -9,7 +9,7 @@ software.
 
 1. Definitions
 
-The terms "reproduce," "reproduction," "derivative works," and "distribution"
+The terms "reproduce", "reproduction", "derivative works" and "distribution"
```

Comment:
The original text, with the commas inside the quotation marks, is technically correct:
http://www.grammarbook.com/punctuation/quotes.asp (see rule 3a)
 
As a programmer, this part of English, with its unbalanced delimiters, makes me highly uncomfortable. But that's the language we're stuck with.